params.yml

```diff
@@ -0,0 +1,130 @@
+
+zkevm_l2_admin_address: "0xE34aaF64b29273B7D567FCFc40544c014EEe9970"
+zkevm_l2_admin_private_key: "0x12d7de8621a77640c9241b2595ba78ce443d05e94090365ab3bb5e19df82c625"
+
+zkevm_l2_loadtest_address: "0x81457240ff5b49CaF176885ED07e3E7BFbE9Fb81"
+zkevm_l2_loadtest_private_key: "0xd7df6d64c569ffdfe7c56e6b34e7a2bdc7b7583db74512a9ffe26fe07faaa5de"
+
+zkevm_l2_agglayer_address: "0x351e560852ee001d5D19b5912a269F849f59479a"
+zkevm_l2_agglayer_private_key: "0x1d45f90c0a9814d8b8af968fa0677dab2a8ff0266f33b136e560fe420858a419"
+
+zkevm_l2_dac_address: "0x5951F5b2604c9B42E478d5e2B2437F44073eF9A6"
+zkevm_l2_dac_private_key: "0x85d836ee6ea6f48bae27b31535e6fc2eefe056f2276b9353aafb294277d8159b"
+
+# Keystore password.
+zkevm_l2_keystore_password: pSnv6Dh5s9ahuzGzH9RoCDrKAMddaX3m
+
+# Database configuration.
+zkevm_db_postgres_port: 5432
+
+# AggLayer db
+zkevm_db_agglayer_hostname: agglayer-db
+zkevm_db_agglayer_name: agglayer_db
+zkevm_db_agglayer_user: agglayer_user
+zkevm_db_agglayer_password: PzycR2uB6PQv8ahj465ExvdyRLkknRNW
+
+# Bridge db
+zkevm_db_bridge_hostname: bridge-db
+zkevm_db_bridge_name: bridge_db
+zkevm_db_bridge_user: bridge_user
+zkevm_db_bridge_password: aXPqaRvgo5DfnTbHtpYS9rMhVpjvb6tY
+
+# DAC db
+zkevm_db_dac_hostname: dac-db
+zkevm_db_dac_name: dac_db
+zkevm_db_dac_user: dac_user
+zkevm_db_dac_password: PzycR2uB6PQv8ahj465ExvdyRLkknRNW
+
+# Event db
+zkevm_db_event_hostname: event-db
+zkevm_db_event_name: event_db
+zkevm_db_event_user: event_user
+zkevm_db_event_password: rJXJN6iUAczh4oz8HRKYbVM8yC7tPeZm
+
+# Pool db
+zkevm_db_pool_hostname: pool-db
+zkevm_db_pool_name: pool_db
+zkevm_db_pool_user: pool_user
+zkevm_db_pool_password: Qso5wMcLAN3oF7EfaawzgWKUUKWM3Vov
+
+# Prover/Executor db
+zkevm_db_prover_hostname: prover-db
+zkevm_db_prover_name: prover_db
+zkevm_db_prover_user: prover_user
+zkevm_db_prover_password: SR5xq2KZPgvQkPDranCRhvkv6pnqfo77
+
+# State db
+zkevm_db_state_hostname: state-db
+zkevm_db_state_name: state_db
+zkevm_db_state_user: state_user
+zkevm_db_state_password: rHTX7EpajF8zYDPatN32rH3B2pn89dmq
+
+# L1 configuration.
+l1_network_id: 2718281828459045
+l1_preallocated_mnemonic: code code code code code code code code code code code quality
+l1_rpc_url: http://el-1-geth-lighthouse:8545
+l1_ws_url: ws://el-1-geth-lighthouse:8546
+# TODO: If we need to fund the aggregator, sequencer, and admin accounts for posting batches.. How much should we prefund??
+l1_funding_amount: 100ether
+
+# L2 configuration.
+zkevm_deployment_salt: "0x0000000000000000000000000000000000000000000000000000000000000001"
+zkevm_fork_id: 8
+zkevm_test_deployment: true
+zkevm_l2_chain_id: 101010101
+zkevm_rollup_consensus: PolygonValidiumEtrog # PolygonZkEVMEtrog
+zkevm_da_protocol: PolygonDataCommittee
+zkevm_rollup_description: kurtosis devnet
+zkevm_rollup_name: kurtosis-test
+zkevm_rollup_real_verifier: false
+
+# Permissionless node configuration.
```

Comment:
Question: Should we move the permissionless-node configuration to its own file?

Keeping the settings for both cdk and permissionless-node packages is fine because we share a lot of settings, but it can be overwhelming for someone who needs to navigate the configuration just to start a permissionless node. I would prefer everything to be in one place, but I would be happy to hear what you think.